src/Symfony/Component/FeatureFlags/Strategy/RequestQueryStrategy.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\FeatureFlags\Strategy;
+
+use Symfony\Component\FeatureFlags\StrategyResult;
+
+// TODO: make it case insensitive ?
+final class RequestQueryStrategy implements StrategyInterface
+{
+    public function __construct(
+        private readonly string $queryParameterName,
+    ) {
+    }
+
+    public function compute(): StrategyResult
+    {
+        if (!array_key_exists($this->queryParameterName, $_GET)) {
```

Comment:
I've not read all the PR comments, yet
but is there any reason this $_GET is used and not the http component as wrapper?

Reply:
This one is part of the component. But in the bundle it is wrapped with the http foundation component. I am not sure to keep the one with $_GET but I find it easier to use outside the framework as a whole